src/model/document.js

```diff
@@ -213,7 +218,7 @@ export default class Document {
 	 * @returns {Boolean}
 	 */
 	hasRoot( name ) {
-		return this.roots.has( name );
+		return !!this.roots.get( name );
```

Comment:
Can we remove `hasRoot` method and replace its usage with `getRoot` (which should not throw but return falsy value if there is no such root? I feel that now when the collection does not have `has` method this code is ridiculous.